lib/components/c_facets/Drag.js

```diff
@@ -84,8 +84,8 @@ function Drag$start() {
     });
 
     this.owner.onMessages({
-        'getstatestarted': _removeDragAttribute,
-        'getstatecompleted': _addDragAttribute
+        'getstatestarted': _removeDragAttribute.bind(this),
```

Comment:
Why are you binding to `this`, all subscribers in milo will implicitly bind to whatever the `on` was called on as default, then you can change context using `{context: blah, subscriber: func}`